lib/linter.js

```diff
@@ -178,20 +178,24 @@ var runChecks = function (ast, fullPath, lines, config) {
     return results;
 };
 
-var mergeLinters = function (defaultLinters, config) {
+var mergeLintersFromConfig = function (defaultLinters, config) {
```

Comment:
any reason for the fn name change? thought it was pretty clear before what the fn did, so I'm curious :)